versioned_docs/version-2.18/reference/global-options.mdx

```diff
@@ -317,9 +317,9 @@ ignore_warnings = [
 
 Use this Pants version. Note that Pants only uses this to verify that you are using the requested version, as Pants cannot dynamically change the version it is using once the program is already running.
 
-If you use the `scie-pants-linux-x86_64` script from https://www.pantsbuild.org/v2.18/docs/installation, however, changing the value in your `pants.toml` will cause the new version to be installed and run automatically.
```

Comment:
This change looks like a good one. I think I'll make a new issue for this and a new PR.